tools/grift/grpc_plugins_generator.patch

```diff
@@ -0,0 +1,2417 @@
+From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
```

Comment:
I don't think this is a bad interim solution, yet ideally, we would submit them upstream. It would quickly become un-maintainable however if this isn't pushed upstream, that is.